solidity/dashboard/src/reducers/liquidity-rewards.js

```diff
@@ -16,6 +16,7 @@ const initialState = {
   KEEP_ETH: { ...liquidityPairInitialData },
   TBTC_ETH: { ...liquidityPairInitialData },
   KEEP_TBTC: { ...liquidityPairInitialData },
+  KEEP: { ...liquidityPairInitialData },
```

Comment:
I don't really like the idea of the state name for keep only pool to be just "KEEP" - seems too generic. Maybe we could rename it to KEEP_ONLY? 🤔